src/Microsoft.ServiceFabric.Client/IImageStoreClient.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.ServiceFabric.Client
     /// <summary>
     /// Interface containing methods for performing ImageStoreClient operataions.
     /// </summary>
-    public partial interface IImageStoreClient
+    public interface IImageStoreClient
```

Comment:
Keep thsi file as IImageStoreClient_.cs, you can add only UploadApplicationPackageAsync to this.
Leave UploadFileChunkAsync UploadFileAsync in ImageStoreClient.cs

Reply:
Sure, feel free to suggest some other suffix :)

Reply:
I think it would be much more easily understood if the generated files and the extensions were kept in different folders. this is probably out of scope for this PR though

Reply:
yes, lets not mobie to a folder for now, feel free to choose some other suffix.